src/js/libcs/RenderBackends.js

```diff
@@ -868,3 +868,9 @@ globalInstance.exportPDF = function() {
         exportWith(PdfWriterContext, wnd);
     });
 };
+
+globalInstance.exportPNG = function() {
+    var data = csctx.canvas.toDataURL();
```

Comment:
Personally I was wondering how this defines the format to be PNG. I then [read](https://developer.mozilla.org/en-US/docs/Web/API/HTMLCanvasElement/toDataURL) that PNG is the default format, but I'd rather make that explicit by passing `"image/png"` as an argument to that function. That makes it easier to undertsand what's happening, and also to adapt it to other formats if we ever need those.

All the other exporting functions are using blobs instead of data URLs. I wonder whether there is any reason to prefer one over the other in the situation at hand. A data url is far longer, but also more portable and more permanent. So perhaps fine for this task unless we run into URL length problems in some real world setup.